torchaudio/transforms.py

```diff
@@ -155,6 +155,13 @@ class GriffinLim(torch.nn.Module):
             Values near 1 can lead to faster convergence, but above 1 may not converge. (Default: ``0.99``)
         length (int, optional): Array length of the expected output. (Default: ``None``)
         rand_init (bool, optional): Initializes phase randomly if True and to zero otherwise. (Default: ``True``)
+
+        Example
+            >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
+            >>> transformgri = transforms.GriffinLim(n_fft=800)
+            >>> transformspec = transforms.Spectrogram(n_fft=800)
+            >>> x = transformspec(waveform)
+            >>> transgriffinlim = transformgri(x)
```

Comment:
Let's not add ``transforms.Spectrogram`` in the example since it returns the complex-type tensor by default. GriffinLim requires the magnitude of the spectrogram which has real type.
Also the Example has indentation issue. Could you make changes in this way:
```suggestion
    Example
        >>> batch, freq, time = 2, 257, 100
        >>> spectrogram = torch.randn(batch, freq, time)
        >>> transform = transforms.GriffinLim(n_fft=512)
        >>> waveform = transform(spectrogram)
```